build.cake

```diff
@@ -236,14 +236,17 @@ Task("Run-MSBuild-UnitTests")
 
 Task("Generate-CMake")
     .WithCriteria(() => IsRunningOnUnix())
-    .IsDependentOn("Build-NetCore")
     .Does(() =>
     {
         var cmakeSettings = new CMakeSettings
         {
             Generator = "Unix Makefiles",
             OutputPath = $"{CMakeBuildDir}",
             SourcePath = ".",
+            Options = new[]
+            {
+                $"-DCMAKE_BUILD_TYPE={CMakeConfiguration}",

```

Comment:
Copied from the `Makefile`.
The config dependent `OutputPath` was already being used by the `Build-CMake` and `Test-CMake` targets below.